src/backend/database_models/conversation.py

```diff
@@ -34,6 +52,13 @@ class Conversation(Base):
     def messages(self):
         return sorted(self.text_messages, key=lambda x: x.created_at)
 
+    @property
+    def file_ids(self):
```

Comment:
Great usage of property!

Reply:
so cool that sqlalchemy orm can do this!